### ISSUE TITLE
[Extension] Unable to delete node group like namspace, pod in first attempt

### ISSUE THREAD

**Issue by Yashsharma1911** [meshery/meshery#10751]
### Current Behavior
When we delete any node group we are unable to delete it in first attemt, on first clicking on delete it changes the node group icon and if I delete it again, then it deletes the node group

It might be because when we drop a node group we drop two components, one is node group itself and one is square inside the node group act as inventory wallet, first it delete the inventory wallent and then node group component, also on pressing the DELETE key it first delete the inventory wallet then node group

Find attached video for issue

https://github.com/meshery/meshery/assets/71271069/16908d7a-ea39-4580-b4a5-85583c43efd5

### Environment

- **Host OS:** Mac Linux Windows
- **Platform:** Docker or Kubernetes
- **Meshery Server Version:** stable-v
- **Meshery Client Version:** stable-v

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

**Comment by Yashsharma1911** [comment]
// @captain-Akshay

**Comment by aabidsofi19** [comment]
yup , it deletes the wallet first , this is part of the same issue i mentioned on the call
